src/fillLocalStorage.tsx

```diff
@@ -0,0 +1,26 @@
+const Users = [
```

Comment:
```suggestion
const USERS = [
```
в PascalCase именуются либо классы, либо компоненты